content/post/hypriotos-barbossa-for-raspberry-pi-3.md

```diff
@@ -0,0 +1,109 @@
+
+
+## Updated Docker and Docker-Tools
+Compared to our latest HypriotOS Berry 0.7 we did upgrade the Docker-Engine to v1.11 which now has several binaries instead of one.
+Besides the original Docker binary there is now also a Containerd and a Runc binary. Containerd is basically a supervisor for individual containers that are run by Runc.
+
+The main reasons for splitting up the Docker binary were the efforts to support the [standardization of container technology](https://www.opencontainers.org/) and to make it easier to maintain the existing technology stack.
+You can get a good overview about these changes in the [blog post](https://blog.docker.com/2016/04/docker-engine-1-11-runc/) announcing the release of Docker 1.11.  
+
+And there is also a very intersting deep dive into the details of those changes: 
+
+<iframe id="ytplayer" type="text/html" width="1000" height="560" src="http://www.youtube.com/embed/QL8F2MLCybo" frameborder="0"></iframe>
+
+## Full Raspberry Pi 3 Support
+Noteworthy is also that we now have full support for the Raspberry Pi 3.
+
+With the help of our [flash-tool](https://github.com/hypriot/flash) it is really easy to bring a Raspberry Pi 3 online with only WiFi network connectivity:
+
+```
+flash --ssid MyNetworkName --password SomeSecret https://github.com/hypriot/image-builder-rpi/releases/download/v0.8.0/sd-card-rpi-v0.8.0.img.zip
+```
+
+After that you can usually connect to your Pi by:
+
+```
+ssh pirate@black-pearl
+```
+
+The password is 'hypriot'. And that's it.
+
+By the way this also works if you attach an external WiFi adapter to a Raspberry Pi Zero, 1 and 2.
+
+## Customizing your Raspberry Pi Configuration
+The ability to customize the hostname of your Pi, to add WiFi credentials, etc. is made possible with the help of device-init.
+
+In previous versions we used a tool called [occi](https://github.com/adafruit/Adafruit-Occi) from Addafruit to allow hostname and WiFi customization.
+
+In this release we removed it and replaced it with device-init.
+
+device-init is a small programm that is started when your Raspberry Pi boots and it allows to customize a couple of settings.
+It takes its configuration from a file called 'device-init.yaml' which is located in the /boot directory. 
+
+This /boot directory is basically a small FAT partition on the SD cards that contains HypriotOS.
+It can be easily accessed from your host computer before you boot your Pi. You can edit the 'device-init.yaml' file directly with a text-editor or by using the flash-tool as demonstrated above.
+
+We are going to add more features - like adding public ssh-keys, setting the locale or timezone - to device-init step by step in the next weeks.
+
+## Hypriot Cluster-Lab
+We added the Hypriot Cluster-Lab to the image, too.
+
+It is disabled by default, but can easily be enabled when you flash the image:
+
+```
+flash --cluster-lab true https://github.com/hypriot/image-builder-rpi/releases/download/v0.5.6/sd-card-rpi-v0.5.6.img.zip
```

Comment:
@Govinda-Fichtner Now as I see an example of the `--cluster-lab` option I think we could remove the `true` as there is only one option that flash can set. Without that option the cluster lab is disabled by default. Just the option that turns the feature on. WDYT?